tests/phpunit/tests/functions/forceSslAdmin.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Tests for the force_ssl_admin function.
+ *
+ * @group functions.php
+ *
+ * @covers ::force_ssl_admin
+ */#
+class Tests_Functions_ForceSslAdmin extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 57261
+	 */
+	public function test_force_ssl_admin() {
+		$default = force_ssl_admin();
+		$this->assertFalse( $default, 'default' );
```

Comment:
The `message` parameter should describe what went wrong if the assertion fails. It shouldn't be describing what you're doing.